src/ProcessMaker/Nayra/Bpmn/InclusiveGatewayTrait.php

```diff
@@ -90,6 +104,13 @@ protected function buildConditionedConnectionTo(FlowNodeInterface $target, calla
             );
         }
         $outgoingTransition->attachEvent(TransitionInterface::EVENT_AFTER_CONSUME, function(TransitionInterface $transition)  {
+
+            foreach ($this->getOwnerProcess()->getInstances()->toArray() as $instance) {
```

Comment:
Check if the second parameter of the callback receives the token or tokens consumed